Voice Note/Views/BottomBarView.swift

```diff
@@ -119,7 +123,35 @@ struct BottomBarView: View {
             )
         }.ignoresSafeArea(.all)
     }
-    
+    func saveVoiceNote() {
+        guard let url = voiceNoteViewModel.fileUrlList.last else {
+            return
+        }
+
+        let newVoiceNote = VoiceNote(context: managedObjectContext)
+            newVoiceNote.id = UUID()
+            newVoiceNote.text = speechRecognizer.transcriptionText
+            newVoiceNote.title = "Note Title"
```

Comment:
how about first few words from transcription to use as a Title? @giaongo @TaiNguyen2407
`newVoiceNote.title = speechRecognizer.transcriptionText.prefix(12)+"..."`

Reply:
Result would be like
### `This is my vo...`

Reply:
@anwar-ulhaq  Prefix part for title might make the UI look confusing when both  ListItem / RecordingList shares the similar text of the transcription and title on the same screen. If you would like to make a prefix for title, I suggest that we only display prefixed title in the RecordingListView without the full transcription text? Full transcription text only shows when user clicks to see the DetailView?